Source/Initialization/PlasmaInjector.cpp

```diff
@@ -118,8 +118,13 @@ PlasmaInjector::PlasmaInjector (int ispecies, const std::string& name)
         mass = species::get_mass( physical_species );
     }
 
-    std::string s_inj_style;
-    pp_species_name.query("injection_style", s_inj_style);
+    // Parse injection style
+    std::string injection_style;
+    pp_species_name.query("injection_style", injection_style);
+    std::transform(injection_style.begin(),
```

Comment:
`query`s are optional. You might want to check the return value before attempting a transform?
Although on the other hand, `std::string` probably just empty if this fails and then the transform goes over an empty range...?

Reply:
I guess this should not be a problem anymore, with the new default `injection_style = "none"`.